assets/tests/components/__snapshots__/modal.test.tsx.snap

```diff
@@ -143,10 +153,6 @@ exports[`Modal renders loading modal when appropriate 1`] = `
         Loading...
       </div>
     </div>
-    <div
-      aria-hidden="true"
-      class="c-modal-backdrop"
-    />
```

Comment:
The [new backdrop](https://github.com/mbta/skate/pull/2571/files#diff-a5d8b215b8b92f35520456915b39141b327d3f33bf71425344cbb88248bb3f8fR129-R131) seems to be missing `aria-hidden=true`?

Reply:
Ooooooo! Ok fantastic catch. This is important and I'm surprised Restart UI doesn't have the nice aria features built-in.  Do you know why we moved towards using Restart UI as opposed to component libraries that maybe have more of the accessibility framework? It's been a while since I've worked on keeping an app accessible, so it might be so easy to incorporate that we don't care to have a library do it, but I would guess that it'd be nice to let a library do some of the lifting?

Reply:
IIRC We want to move to Restart UI to get rid of the management code of opening and closing a modal and managing focus states without needing to fully turn ourselves over to a specific modal yet. I'm not sure why the library doesn't add `aria-hidden` here, I'm assuming there may be some info I'm not aware of and maybe aria-hidden isn't needed, but I assume that since we're in control of how the backdrop is rendered it's up to us and not really the library.

Reply:
That's fair. I checked the accessibility tree (had to learn how to do that) and sure enough, the backdrop is not getting hidden at all, so great catch, and easy enough inline fix